grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -599,7 +599,7 @@ final void maybeLogRequestContent(@Nullable Object message) {
         }
     }
 
-    final void maybeLogFailedRequestContent(@Nullable Throwable cause) {
+    public final void maybeLogFailedRequestContent(@Nullable Throwable cause) {
```

Comment:
```suggestion
    private void maybeLogFailedRequestContent(@Nullable Throwable cause) {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!